docs/source/user_guide/runtime-image-conf.md

```diff
@@ -112,21 +117,25 @@ $ elyra-metadata remove runtime-images \
 The runtime image configuration properties are defined as follows. The string in the headings below, which is enclosed in parentheses, denotes the CLI option name.
 
 ##### Name (display_name)
+
 A user-friendly name for runtime image configuration. This property is required.
 
 Example: `My runtime image`
 
 ##### Description (description)
+
 Description for this runtime image configuration.
 
-Example: `My custom runtime Docker image`
+Example: `My custom runtime container image`
 
 ##### Image Name (image_name)
-The name and tag of an existing Docker image in the public Docker Hub container registry that meets the stated prerequisites. This property is required.
+
+The name and tag of an existing container image in a public container registry that meets the stated prerequisites. This property is required.
```

Comment:
Can you please mention the optional registry component `registry/owner/image:tag` and clarify that if no registry is specified, docker.io is used as default.

Reply:
Updated in 453592bbaec142ba2e98f1e548e39df2546be820